pages/en/lb3/Redis-connector.md

```diff
@@ -67,7 +67,7 @@ Edit `datasources.json` to add other properties that enable you to connect the
     <tr>
       <td>host</td>
       <td>String</td>
-      <td>Database host name</td>
+      <td>Database host name. When this property is set, also the property **port** needs to be set.</td>
```

Comment:
We call `redis.createClient(port, host, options)`. See https://github.com/NodeRedis/node_redis#rediscreateclient. I think this line can be made more robust - https://github.com/strongloop-community/loopback-connector-redis/blob/master/lib/redis.js#L37. 

@TwistedLogic Do you want to fix the code?

Reply:
I already submitted a PR.

Reply:
Got it - https://github.com/strongloop-community/loopback-connector-redis/pull/57

Reply:
I'd rephrase this to say `When this property is set, the port property **must also** be set.` or something to that effect.